charts/digibyte/templates/ingress-web.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: digibyte-node-ingress-web
```

Comment:
Can you please modify the resources names to include the release name to allow multiple deployments without conflicts in digibyte and dash?

Reply:
OK

Reply:
Should be fixed with https://github.com/threefoldtech/vdc-solutions-charts/commit/7c244eda752e81b65784262e3ec7a6721b081227